wp-content/themes/csisnuclearnetwork/template-parts/block-post-related.php

```diff
@@ -13,14 +13,21 @@
 
 ?>
 <article <?php post_class('post-block-related'); ?> id="post-<?php the_ID(); ?>">	
-		<?php if ( has_post_thumbnail() ) : ?>
-			<a href="<?php the_permalink(); ?>" class="post-block-related__img" title="<?php the_title_attribute(); ?>">
-				<?php the_post_thumbnail( 'large' ); ?>
-			</a>
-		<?php endif; ?>
-
+	<h4 class='post-block-related__subtitle'><a class="text-strong">ANALYSIS/</a><span><?php nuclearnetwork_display_categories() ?></span></h4>
 	<?php
-		nuclearnetwork_posted_on('M j, Y');
-		the_title( '<h3 class="post-block-related__title"><a class="post-block-related__title" href="' . esc_url( get_permalink() ) . '">', '</a></h3>' );
+		the_title( '<h3 class="post-block-related__title text-stong"><a class="post-block-related__title" href="' . esc_url( get_permalink() ) . '">', '</a></h3>' );
 	?>
+	<div class='post-block-related__byline'>
+		<span class="post-block-related__byline-date"> 
+			<?php nuclearnetwork_posted_on('M j, Y') ?> 
+		</span>
+		<span class="post-block-related__byline-divider">|</span>
+		By 
+		<span class="post-block-related__byline-authors">
```

Comment:
We don't need to add this extra `span`, as the author is already in a definition list.